component/portal/src/main/resources/db/changelog/portal-rdbms.db.changelog-1.0.0.xml

```diff
@@ -329,4 +329,17 @@
     <createSequence sequenceName="SEQ_PORTAL_PERMISSIONS_ID" startValue="1" />
   </changeSet>
 
-</databaseChangeLog>
\ No newline at end of file
+  <changeSet author="portal" id="1.0.0-30" dbms="postgresql">
+    <preConditions onFail="MARK_RAN">
+      <sqlCheck expectedResult="bytea">
+        SELECT DATA_TYPE FROM information_schema.columns where table_name = 'portal_windows' and column_name = 'customization'
+      </sqlCheck>
+    </preConditions>
+    <sql>
+      CREATE TABLE PORTAL_WINDOWS_lo AS SELECT id, title, icon,description,show_info_bar,show_app_state,show_app_mode,theme,width,height,properties,app_type,content_id, lo_from_bytea(0, customization) customization FROM PORTAL_WINDOWS;
+      DROP TABLE PORTAL_WINDOWS;
```

Comment:
Could we just copy the content of the old column in a new column and then just drop the column instead of dropping the whole table, I think it will be safer and will take less time , WDYT ?

Reply:
You're, updated